packages/packages.go

```diff
@@ -1,15 +1,16 @@
-/*
-Copyright 2017 Google Inc. All Rights Reserved.
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-    http://www.apache.org/licenses/LICENSE-2.0
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+//  Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
We should probably fix the year to the current one.

Reply:
Should we, i had this question in my mind, should we change year or not?

Reply:
I think that we should with each update.